### ISSUE TITLE
Winget Package Manager Support

### ISSUE THREAD

**Issue by jctemp** [tectonic-typesetting/tectonic#976]
Tectonic has support for various package manager. Chocolatey and Scoop are great but not by Microsoft. As Microsoft has an own package manager `winget`, it would be an fantastic addition to add Tectonic to the repository. In addition, as the setup process can be difficult, they have release wingetcreate `https://github.com/microsoft/winget-create` to make submission easier by removing the hustle to write a manifest file.
Note: The current installation guide for windows requires to use an external package manager or a powershell script which one has to trust.

Kindest regards

P.S. Haven't found any guidelines where to put this so here I am :)

**Comment by pkgw** [comment]
Agreed that this would be good to support! I don't personally have the bandwidth to look into this, but I am happy to try to assist if someone else wants to take this project on.
